fetch/src/2_process.r

```diff
@@ -0,0 +1,22 @@
+get_data <- function() {
+  # Get the data from ScienceBase
+  mendota_file <- file.path(project_output_dir, 'model_RMSEs.csv')
+  item_file_download('5d925066e4b0c4f70d0d0599', names = 'me_RMSE.csv', destinations = mendota_file, overwrite_file = TRUE)
+}
+  
+    prep_data <- function() {
+      # Prepare the data for plotting
+      eval_data <- readr::read_csv(mendota_file, col_types = 'iccd') %>%
```

Comment:
Hmm, looks like `mendota_file` isn't defined within this function.. how will the function know where to find the raw data that is meant to be read in?

Reply:
Yes, your right!